### PR TITLE
Update dependencies and normalize package.json.

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
-node_modules
-tmp
 .DS_Store
-
+/node_modules/
```
Comment (dayvson) [inline .gitignore]: Why do we need this slash?

File: package.json [hunk 2]
```diff
@@ -3,46 +3,39 @@
...
-  "author": {
-    "name": "Maxwell Da Silva",
-    "email": "dayvson@gmail.com"
-  },
+  "author": "Maxwell Da Silva <dayvson@gmail.com>",
+  "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "https://github.com/dayvson/grunt-static-inline"
+    "url": "git+https://github.com/dayvson/grunt-static-inline.git"
```
Comment (dayvson) [inline package.json]: this url is wrong it should be:
git://github.com/dayvson/grunt-static-inline.git

File: .gitignore (hunk 1 above)
Comment (XhmikosR) [inline .gitignore]: Both work, but the one with the slash is clear it's a folder in root.

File: package.json (hunk 2 above)
Comment (XhmikosR) [inline package.json]: Nope, it's right. It's the default scheme npm is using. Try doing `npm init` in your master and see.

File: package.json [hunk 3]
```diff
@@ -3,46 +3,39 @@
...
-  "main": "Gruntfile.js",
   "engines": {
-    "node": ">= 0.8.0"
+    "node": ">=0.10.0"
   },
-  "preferGlobal": true,
-
   "scripts": {
-    "test": "node node_modules/grunt-cli/bin/grunt test"
+    "test": "grunt test"
```
Comment (dayvson) [inline package.json]: this assumes that you have grunt install globally which I don't think is a good idea.

File: .gitignore (hunk 1 above)
Comment (dayvson) [inline .gitignore]: Everybody knows that node_modules is a folder and this is just the gitignore. 
If you remove these changes and if the commented that I made about the grunt test on line 19 I will merge this pull request

File: package.json (hunk 3 above)
Comment (XhmikosR) [inline package.json]: Nope it doesn't assume that. Node looks locally too.

**Comment by dayvson** [comment]
Thanks a lot @XhmikosR  the project looks much nicer with your contributions :beers:

File: .gitignore (hunk 1 above)
Comment (XhmikosR) [inline .gitignore]: The comment you made for grunt is moot :P

As for this, I guess it's a personal taste. I'll change this one.

**Comment by XhmikosR** [comment]
Cheers :)

**Comment by XhmikosR** [comment]
So any news about this? Would be nice to have a new release with all these fixes (don't forget the tag too)

**Pull request merged**